hypothesis-python/tests/cover/test_core.py

```diff
@@ -23,21 +23,20 @@
 import hypothesis.strategies as s
 from hypothesis import find, given, reject, settings
 from hypothesis.errors import NoSuchExample, Unsatisfiable
+from tests.common.debug import minimal
```

Comment:
I think all the changes in this file are from an unrelated PR?

Reply:
Whoops, yeah - they belong in the deprecation-of-`find` PR, not this one.